lib/trento_web/controllers/v1/host_controller.ex

```diff
@@ -186,14 +186,15 @@ defmodule TrentoWeb.V1.HostController do
       unprocessable_entity: OpenApiSpex.JsonErrorResponse.response()
     ]
 
-  def request_operation(%{assigns: %{host: host}} = conn, %{
-        operation: "saptune_solution_apply"
-      }) do
+  def request_operation(%{assigns: %{host: host}} = conn, %{operation: operation})
+      when operation in ["saptune_solution_apply", "saptune_solution_change"] do
     %{solution: solution} = OpenApiSpex.body_params(conn)
     %{id: host_id} = host
 
     with {:ok, operation_id} <-
-           Hosts.request_operation(:saptune_solution_apply, host_id, %{solution: solution}) do
+           conn
+           |> get_operation()
```

Comment:
In the `cluster_maintenance_change` code I'm working in, I have updated the `operations_policy_plug.ex` code (the `call` function) to assign the operation to `:operation`, so we can get it in the `assign` here, and we don't need to be converting more than once.
I think we can leverage that. If you want I can cherry-pick the change here as well.

Reply:
It's a good idea. While going through the code I was also thinking about that. Thanks!

Reply:
I will modify this in my `cluster_maintenance_change` PR. It is one line change, so it is fine as i